pkg/restclient/volume.go

```diff
@@ -171,3 +171,10 @@ func (c *Client) EditVolume(volname string, req api.VolEditReq) (api.VolumeEditR
 	err := c.post(url, req, http.StatusOK, &resp)
 	return resp, err
 }
+
+// VolumeReset resets volume options
+func (c *Client) VolumeReset(volname string, req api.VolOptionResetReq) error {
+	url := fmt.Sprintf("/v1/volumes/%s/options", volname)
+	err := c.del(url, req, http.StatusOK, nil)
```

Comment:
delete should return 204 status code. instead of returning 200 with empty response body.